safe_transaction_service/history/indexers/proxy_factory_indexer.py

```diff
@@ -50,13 +51,18 @@ def contract_events(self) -> list[ContractEvent]:
         proxy_factory_v_1_4_1_contract = get_proxy_factory_V1_4_1_contract(
             self.ethereum_client.w3
         )
+        proxy_factory_v_1_5_0_contract = get_proxy_factory_V1_5_0_contract(
+            self.ethereum_client.w3
+        )
         return [
             # event ProxyCreation(Proxy proxy)
             proxy_factory_v1_1_1_contract.events.ProxyCreation(),
             # event ProxyCreation(GnosisSafeProxy proxy, address singleton)
             proxy_factory_v1_3_0_contract.events.ProxyCreation(),
             # event ProxyCreation(SafeProxy indexed proxy, address singleton)
             proxy_factory_v_1_4_1_contract.events.ProxyCreation(),
+            # event ProxyCreation(SafeProxy indexed proxy, address singleton)
+            proxy_factory_v_1_5_0_contract.events.ProxyCreation(),
```

Comment:
Shouldn't be necesary as it has the same signature as 1.4.1, but good to be here for readability